admin_guide/managing_networking.adoc

```diff
@@ -420,50 +421,82 @@ $ oc create -f <replication_controller>.json
+it in all other projects that have been
+xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made
+global]. 
+
+[NOTE]
+====
+Multicast global projects are not "global", but instead communicate with only
+other global projects via multicast, not with all projects in the cluster, as is
+the case with unicast.
 ====
 
-If you have <<joining-project-networks,joined networks together>> with
-`oadm pod-network` you will need to enable multicast in each of the projects'
-NetNamespaces in order for it to take effect in any of the projects. You cannot
-enable multicast in the "default" project or in any other project that has been
-<<making-project-networks-global,made global>>, nor can you send multicast
-packets into or out of global projects.
+[[admin-guide-networking-networkpolicy]]
+== Enabling NetworkPolicy
 
-[[networkpolicy]]
-== NetworkPolicy
+[IMPORTANT]
+====
+Enabling the Kubernetes NetworkPolicy is a Technology Preview only. Technology
+Preview features are not supported with Red Hat production service level
+agreements (SLAs), might not be functionally complete, and Red Hat does not
+recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/. 
+====
 
 Kubernetes NetworkPolicy is not currently fully supported by {product-title},
-and the *ovs-subnet* and *ovs-multitenant* plugins ignore NetworkPolicy objects.
-However, a Tech Preview of NetworkPolicy support is available by using the
-*ovs-networkpolicy* plugin.
-
-In a cluster xref:../../install_config/configuring_sdn.adoc#install-config-configuring-sdn[configured to use the *ovs-networkpolicy* plugin],
-network isolation is controlled entirely by NetworkPolicy objects and the associated
-Namespace annotation, as described in the link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network-policy.md[NetworkPolicy documentation].
-In particular, by default, all projects are able to access pods in all other
-projects. A project that wants to be isolated must first opt in to isolation by
-setting the proper annotation on its Namespace object, and then create NetworkPolicy
-objects indicating what incoming connections should be allowed.
+and the *ovs-subnet* and *ovs-multitenant* plug-ins ignore NetworkPolicy
+objects. However, a Tech Preview of NetworkPolicy support is available by using
+the *ovs-networkpolicy* plug-in.
+
+In a cluster
+xref:../../install_config/configuring_sdn.adoc#install-config-configuring-sdn[configured
+to use the *ovs-networkpolicy* plug-in], network isolation is controlled
+entirely by NetworkPolicy objects and the associated Namespace annotation, as
```

Comment:
suggestion: Drop the "as described in..." and make "namespace" a link to the resource instead.